packages/roosterjs-content-model-core/lib/coreApi/addUndoSnapshot/addUndoSnapshot.ts

```diff
@@ -1,5 +1,13 @@
+import { parseEntityFormat } from 'roosterjs-content-model-dom';
+import type {
+    AddUndoSnapshot,
+    EntityOperationEvent,
+    Snapshot,
+} from 'roosterjs-content-model-types';
 import { createSnapshotSelection } from './createSnapshotSelection';
-import type { AddUndoSnapshot, Snapshot } from 'roosterjs-content-model-types';
+import { getPath } from './getPath';
+
+const ENTITY_INFO_NAME = '_Entity';
```

Comment:
Consider export this const from entityUtils.ts, so we just have one single source of this const

Reply:
And mark the export as `@internal`